src/lib.rs

```diff
@@ -58,6 +76,9 @@ impl Contract {
             authors: UnorderedMap::new(StorageKey::AuthorToAuthorPosts),
             communities: UnorderedMap::new(StorageKey::Communities),
             featured_communities: Vec::new(),
+            last_project_id: 0,
```

Comment:
This info is not necessary and to keep track with the real last project id it's possible to introduce bugs. You can get so with `self.projects.len()`.

Reply:
Ah I see it's because community deletion